likecoin/includes/constant/options.php

```diff
@@ -46,6 +46,7 @@
 define( 'LC_MATTERS_INFO', 'lc_matters_info' );
 define( 'LC_ISCN_DEV_INFO', 'lc_iscn_dev_info' );
 define( 'LC_ISCN_INFO', 'lc_iscn_info' );
+define( 'LC_IMAGE_INFO', 'lc_image_info' );
```

Comment:
`LC_MATTERS_IMAGE_INFO` and `lc_matters_image_info` seems more explicit since we only use this in matters